### DISCUSSION TITLE
Save random playlist

### DISCUSSION THREAD

**Discussion by ElHado** [navidrome/navidrome#2550] [Q&A]
Hi all

Fist: Thanks for this great software!

Is there any way to save a random generated playlist?

**Comment by ElHado** [comment]
Hi all, 

upgraded to 0.50.0 right now but seems not to be an option for this there, too. 

Is this planned for the future? Or is there any option for it already?

**Comment by deluan** [comment]
You can have a Random Smart Playlist (See #1417). It would be regenerated every time you open it (in Navidrome and some Subsonic clients)

**Comment by ElHado** [comment]
Thanks for you reply but I think it's not what I'm searching for. I want to edit the playliste (remove songs I don't want) and save it then. Smart playlists seems like they are not editable.

**Comment by deluan** [comment]
There's no easy way to accomplish what you want to do. One possible option is to have this Random Smart Playlist, export it to a .m3u file and save the file in your Music Folder. On next scan it will be imported as a plain new playlist in the UI, one that you can edit.

I'll take this use case in consideration while rewriting the UI (work in progress).
